modules/apps/web-experience/export-import/export-import-service/src/main/java/com/liferay/exportimport/controller/PortletImportController.java

```diff
@@ -418,7 +418,6 @@ public void importPortletPreferences(
 
 				if (ownerType == PortletKeys.PREFS_OWNER_TYPE_GROUP) {
 					curPlid = PortletKeys.PREFS_PLID_SHARED;
```

Comment:
Not sure which change is being referenced, but judging by your comment, I would say see the above answer.